nestkernel/connector_base.h

```diff
@@ -450,6 +652,25 @@ class Connector< K_cutoff, ConnectionT > : public vector_like< ConnectionT >
     C_[ K_cutoff - 1 ] = c;
   };
 
+  /**
+   * Create a new connector and remove the ith connection
+   * @param Cm1 Original connector of size K_cutoff.
+   * @param i The index of the connection to be deleted.
+   */
+  Connector( const Connector< K_cutoff, ConnectionT >& Cm1, size_t i ) //: syn_id_(Cm1.get_syn_id())
```

Comment:
Please also add some information about the algorithm. See [above](https://github.com/nest/nest-simulator/pull/120/files#r44636702).